src/source/Rtcp/RtcpPacket.c

```diff
@@ -142,3 +142,14 @@ STATUS rembValueGet(PBYTE pPayload, UINT32 payloadLen, PDOUBLE pMaximumBitRate,
     LEAVES();
     return retStatus;
 }
+
+// converts 100ns precision time to ntp time
+UINT64 convertTimestampToNTP(UINT64 time100ns)
```

Comment:
Minor: Macro-fy this? CONVERT_TIMESTAMP_TO_NTP

Reply:
i would prefer to leave it alone - too much logic for a macro

Reply:
Sounds good. This is not used in the tight loop or perf sensitive code path so it's OK to leave as an API call. Most decent compilers would inline this anyway

Reply:
exactly